es6-template-literals/index.html

```diff
@@ -0,0 +1,36 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>Javascript ES6 - Template Literals</title>
+</head>
+<body>
+  <script>
+    const number = 9200;
+    const street = ' Irvine Center Drive';
+    const address = `${number} ${street}`;
```

Comment:
This still isn't an exact conversion:

---

![image](https://user-images.githubusercontent.com/16143771/127086878-8fed7db0-738a-4f6c-a79a-af85726a0bce.png)

---

![image](https://user-images.githubusercontent.com/16143771/127086887-a897d928-a207-46f2-8d5c-6a0995eccdb0.png)